README.md

```diff
@@ -67,15 +77,38 @@ In Swift / Objective-C, Babbage must be initialized with two arguments. The firs
 // In AppDelegate.swift
 let dir = NSFileManager.defaultManager().URLsForDirectory(NSSearchPathDirectory.DocumentDirectory, inDomains: NSSearchPathDomainMask.UserDomainMask).last
 let key = "12345678901234567890123456789012".dataUsingEncoding(NSUTF8StringEncoding)
-MDBabbage.startWithDatastoreAtURL(dir, encryptionKey: key)
+MDBabbage.startWithEnvironment(MDClientEnvironment.Sandbox, apiToken: "Your provided API token", publicDirectory: dir, privateDirectory: dir, encryptionKey: key)
+```
+
+## Self Registering the user
+You can create an account for storing data to AWS S3 buckets.
+
+```swift
+// In CreateAccountViewController.swift
+let userEmail = "unique email address"
+let userPassword = "password complying to have  
+•  At least 8 characters long
+•  At least one upper-case letter
+•  At least one lower-case letter
+•  At least one numeric digit"
+let userSecurityQuestionID = "id of the array element from list of questions"
+let userSecurityAnswer = "answer to the security question"
+
+client.registerSubjectWithEmail(userEmail, password: userPassword, securityQuestionID: userSecurityQuestionID, securityQuestionAnswer: userSecurityAnswer completion:(void (^)(NSError *error))completion
 ```
 
-## Loading Data from the Datastore
+## Loading Data to the Datastore
 You can store and retrieve persistent data using the Datastore class.
 
 ```swift
-let datastore = MDDatastoreFactory.create()
-let user = datastore.userWithID(Int64(self.userID))
+let img = UIGraphicsGetImageFromCurrentImageContext()
+let imageData = UIImageJPEGRepresentation(img, 0.5)
+
+// Collecting the data from the image view
+subject.collectData(imageData, withMetadata: "Random String", withContentType: "image/png", withAppSpecificTag: "", withSchemaURI: "", completion: { (dataEnvelope:  MDSubjectDataEnvelope!, err: NSError!)}
```

Comment:
It looks like the data is JPEG but you're sending `image/png` as the format - we probably want to make that consistent.